src/main/java/com.vite/cli/ViteCLI.java

```diff
@@ -0,0 +1,69 @@
+package com.vite.cli;
+
+import com.vite.service.Client;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.io.OutputStream;
+import java.io.OutputStreamWriter;
+import java.io.PrintWriter;
+
+public class ViteCLI {
+    private final BufferedReader in;
+    private final PrintWriter out;
+    private static final Logger LOGGER = LoggerFactory.getLogger(ViteCLI.class);
+    private final Client client;
+
+    public ViteCLI(InputStream inputStream, OutputStream outputStream, Client client) {
+        LOGGER.debug("Command line interface started...");
+        in = new BufferedReader(new InputStreamReader(inputStream));
+        out = new PrintWriter(new OutputStreamWriter(outputStream));
+        this.client = client;
+    }
+
+    public void run() {
+        for (;;) {
+            out.println("What would you like to do?\n\t1) Ping Server \n\t9) Exit");
+            out.flush();
+            Integer choice = 0;
+            try {
+                choice = Integer.parseInt(in.readLine());
```

Comment:
My preference would be to do something kind of like this:
```
final Map<String, Function<String[], String>> commands = new HashMap<String, Command>();
commands.put("ping", this::ping);
commands.put("quit", this::quit);

String input = null;
while ((input = in.readLine) != null) {
   final String[] inputCommand = input.trim().split(" ");
   if (inputCommand.length == 0) {
       continue;
   }
   final String commandName = command[0];
   final String[] commandArgs = Arrays.copyOfRange(inputCommand, 1, inputCommand.length);
   
   if (!commands.containsKey(commandName) {
       out.println("Unrecognized command: " + commandName);
       out.flush();
       continue;
   }
   out.println(commands.get(commandName).apply(commandArgs));
   out.flush();
}
out.close();
```

Then just define the methods to do stuff:
```
public String ping(String[] args) {
    return client.pingServer();
}
```

Printing help could be done automagically by making the command map an instance variable and using a `Command` class that is constructed with the command help info for each command. Then just add a `"help"` command that prints out that iterates over the map to construct a help string.